### ISSUE TITLE
Goal-by-target layout: targets difficult to read in high-contrast

### ISSUE THREAD

**Issue by brockfanning** [open-sdg/open-sdg#240]
In high-contrast mode, the target names in the "goal-by-target" layout are grey on black, which makes them difficult to read.
